jwql/instrument_monitors/common_monitors/cosmic_ray_monitor.py

```diff
@@ -0,0 +1,727 @@
+#! /usr/bin/env python
+
+"""This module runs the Cosmic Ray Monitor.
+
+This module contains code for the cosmic ray monitor, which currently
+checks the number and magnitude of jumps in all observations performed
+with MIRI and NIRCam. The code first checks MAST for any new MIRI or
+NIRCam observations that have not yet been run through the monitor. It
+then copies those files to a working directory, where they are run
+through the pipeline, and for which the output is stored in a new
+directory for each observation. Each observation is then analyzed for
+jumps due to cosmic rays, of which the number and magnitude are
+recorded. This information is then inserted into the stats database
+table.
+
+Authors
+-------
+
+    - Mike Engesser
```

Comment:
@bhilbert4 don't forget to add yourself here!!

Reply:
Done